drivers/hmis/app/models/hmis/ce/referral_enroller.rb

```diff
@@ -55,11 +50,18 @@ def create_enrollment(message)
 
     # this is not actually an accessor method, even though RuboCop thinks it is
     def set_move_in_date(message) # rubocop:disable Naming/AccessorMethodName
-      project = referral.target_project
-      raise 'access denied' unless message.user.can_edit_enrollments_for?(project)
+      # No need to check permission for editing the enrollment.
+      # If the user can complete this step, they can do its side effects, even if they don't have direct permission
```

Comment:
Highlighting this out-of-scope permissions change for review/discussion - let me know if you agree with the change I've proposed here. (See "Permissions" section of the CE-in-HMIS spec doc)

Reply:
👍  thanks